data-types/datetime64/README.md

```diff
@@ -0,0 +1,33 @@
+# Datetime64 data type
+
+Defines a data type for a datetime object based on a 64-bit integer.
+
+## Permitted fill values
+
+The value of the `fill_value` metadata key must be a signed 64-bit integer.
+
+## Example
+
+For example, the array metadata below specifies that the array uses the datetime64 data type:
+
+```json
+{
+    "data_type": "datetime64",
+    "fill_value": -9223372036854775808,
+    "configuration": {
+      "unit": "s"
+    },
+}
+```
+
+## Notes
```

Comment:
If numpy compatibility is still the design goal for this dtype, I think it would make sense to say something about compatibility with the numpy datetime dtype, e.g. with a link to the [numpy docs ](https://numpy.org/doc/stable/reference/arrays.datetime.html).